charts/gardener/provider-local/extension/doc.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:generate ../../../../hack/generate-controller-registration.sh provider-local . ../../../../VERSION ../../../../example/provider-local/base/controller-registration.yaml DNSProvider:local DNSRecord:local ControlPlane:local Infrastructure:local Network:local OperatingSystemConfig:local Worker:local
+//go:generate ../../../../hack/generate-controller-registration.sh provider-local . '' ../../../../example/provider-local/base/controller-registration.yaml DNSProvider:local DNSRecord:local ControlPlane:local Infrastructure:local Network:local OperatingSystemConfig:local Worker:local
```

Comment:
In general this looks odd. And error-prone - if there is a typo in the VERSION file path it will wrongly assume wrong version/tag, right? 
Anyways `eu.gcr.io/gardener-project/gardener/extensions/provider-local` does not seem to be build and pushed by Gardener CI/CD. WDYT about modifying the `generate-controller-registration.sh` to accept a <version> instead of a <version-file>. This will be a small breaking change for extensions (they will have to pass the VERSION file content instead of the VERSION file path). But I think this approach looks much better and cleaner. We would hard-code the version here  to some stable one for this extension.

Reply:
I have tried your suggestion, however, besides from the downside that it's a breaking change it doesn't seem to make things much better. When I pass `latest` as `VERSION` as you suggested then the `helm package` complains with "Error: Invalid Semantic Version". I could now add two new arguments (chart version and tag instead of version file) but then all "regular" users have to pass the `VERSION` file twice. Do we really want all this? I can do it if you insist, but I doubt that it's much better compared to what is currently proposed, especially w.r.t. the effort adapting all extensions.

Reply:
Probably I am missing details but do we want to always pass `latest`? My idea was to just push for example tag `v0.1.0` and hard-code this version. Do you want it to be `latest` because of e2e tests with prow?

Reply:
OK, you're right, we don't have to use `latest`. It basically doesn't matter what is used since it is always overwritten anyways. Hence, I've addressed your suggestion and used `v0.0.0` for now. OK?